transaction.sql

```diff
@@ -0,0 +1,54 @@
+-- setting the species column to unspecified
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+
+-- Verify 
+SELECT * FROM animals;
+
+--  roll back 
+ROLLBACK;
+
+-- verify 
+SELECT * FROM animals;
+
+-- a transcation
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
```

Comment:
- [ ] Kindly verify that change was made before `COMMIT`.

Reply:
I have done that, but why?